doc/contributing/maintaining-V8.md

```diff
@@ -208,6 +208,62 @@ backport the fix:
 Abandoned V8 branches are supported in the Node.js repository. The fix needs
 to be cherry-picked in the Node.js repository and V8-CI must test the change.
 
+As an example for how to backport changes, consider the bug [RegExp show
+inconsistent result with other browsers](https://crbug.com/v8/5199). From the
+bug we can see that it was merged by V8 into 5.2 and 5.3, and not into V8 5.1
+(since it was already abandoned). Since Node.js `v6.x` uses V8 5.1, the fix
+needed to be backported.
```

Comment:
Let's keep the link on one line, imo it's easier to read.
```suggestion
As an example for how to backport changes, consider the bug
[RegExp show inconsistent result with other browsers](https://crbug.com/v8/5199).
From the bug we can see that it was merged by V8 into 5.2 and 5.3, and not into
V8 5.1 (since it was already abandoned). Since Node.js `v6.x` uses V8 5.1, the
fix needed to be cherry-picked.
```